### PR TITLE
Add notes for building and releasing the documentation.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**
f0b8eea31ba

### CODE REVIEW

File: dev/dev-containers.acbuild.sh [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+
+################################################################################
+# OCI Tools Build
+################################################################################
+
+acbuild run -- go get -d github.com/opencontainers/image-tools/cmd/oci-image-tool
+acbuild run -- go get -d github.com/opencontainers/runtime-tools/cmd/oci-runtime-tool
+
+acbuild run --working-dir /root/go/src/github.com/opencontainers/image-tools -- make all install
```
Comment (passy) [inline dev/dev-containers.acbuild.sh]: Just curious, is `all` as target necessary? Normally `install` implies that.

File: docs/dev.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+::
+
+    ./dev-containers.acbuild.sh
+
+This will make a ``dev-containers.oci`` in the directory. Convert this to
+``dev-containers.aci`` for installation into rkt:
+
+::
+
+    gunzip < dev-containers.oci > dev-containers.oci.tar
```
Comment (passy) [inline docs/dev.rst]: Wait, so `.oci` is actually a `.tar.gz`?

File: docs/releasing.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    pushd containers-gh-pages > /dev/null
+    git checkout -b gh-pages
+
+Clear out everything in the branch. This uses dot globing and extended glob
+options to arrange deletion of everything except the .git directory.
+
+::
+
+    shopt -s dotglob
```
Comment (passy) [inline docs/releasing.rst]: That's some dark magic.

File: docs/dev.rst (hunk 2 above)
Comment (daithiocrualaoich) [inline docs/dev.rst]: Yeah. You'll love the bit in `docs/rkt.rst` of https://github.com/woofwoofinc/containers/pull/12 for converting OCI images to ACI. I kinda gunzipped the OCI but left the tar and docker2aci started working fine. :)

File: docs/releasing.rst (hunk 3 above)
Comment (daithiocrualaoich) [inline docs/releasing.rst]: Not messing around. :) I want those files gone!

File: dev/dev-containers.acbuild.sh (hunk 1 above)
Comment (daithiocrualaoich) [inline dev/dev-containers.acbuild.sh]: I had a look and yes, it is needed in this case. Agree it is unconventional. See the dependencies on the install target in the Makefile at https://github.com/opencontainers/image-tools/blob/master/Makefile#L45. Doesn't include the `all` target and neither is it transitive on the other dependency.

```
# make install
go-md2man -in "man/oci-image-tool.1.md" -out "oci-image-tool.1"
go-md2man -in "man/oci-image-tool-create.1.md" -out "oci-image-tool-create.1"
go-md2man -in "man/oci-image-tool-unpack.1.md" -out "oci-image-tool-unpack.1"
go-md2man -in "man/oci-image-tool-validate.1.md" -out "oci-image-tool-validate.1"
install -d -m 755 /usr/bin
install -m 755 oci-image-tool /usr/bin
install: cannot stat 'oci-image-tool': No such file or directory
Makefile:46: recipe for target 'install' failed
make: *** [install] Error 1
```

**Pull request merged**